### PR TITLE
Blog Post: Second Wave of Education Grants

### PR DESCRIPTION
**Pull request by Arvin21M**
Build Preview:
- [/blog/second-wave-of-education-grants](https://os-website-git-2nd-wave-education-grants-open-sats.vercel.app/blog/second-wave-of-education-grants)

Social Preview:
- image

### CODE REVIEW

**Comment by dergigi** [comment]
On second thought, the whole "wave" of grants stuff might get confusing. We already have waves of nostr grants and waves of bitcoin grants, so I think a different title might work better.

What about "Fruits from the OpenSats Education Initiative" or something along these lines?

**Comment by Arvin21M** [comment]
> On second thought, the whole "wave" of grants stuff might get confusing. We already have waves of nostr grants and waves of bitcoin grants, so I think a different title might work better.
> What about "Fruits from the OpenSats Education Initiative" or something along these lines?

It might be confusing, but the "Education Initiative" designation in the title should alleviate confusion. With that said, these are specifically bitcoin related project grants, but there may come a day where there are nostr related education grants, so perhaps it does make sense to come up with a new recurring title we can use for all future education grant waves. 

I really like the "fruits" title suggestion for this one, but how would we reuse this title for the next wave of education grants, or should each wave of education grant announcements have a new title?

**Comment by dergigi** [comment]
No strong feelings either way, happy to keep the title as is.

File: data/blog/second-wave-of-education-grants.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: "Second Wave of Education Grants"
+date: '2024-09-27'
+tags: ['OpenSats', 'grants', 'bitcoin', 'education', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
```
Comment (dergigi) [inline data/blog/second-wave-of-education-grants.mdx:6]:
```suggestion
authors: ['arvin', 'dergigi', 'dread']
```

**Review by dergigi — COMMENTED** [review]
Looks good to me! ✅

One minor suggestion is to add Dread as an author, as he's the driving force behind the education initiative in the first place.

Other than that I'd say this one's ready for a sanity check by the @OpenSats/volunteers team. :shipit:

**Review by dergigi — APPROVED** [review]
I just did a final re-read and sanity check. All links point to the proper locations, and I didn't find any typos or other issues. 

In other words: LGTM, let's ship it! :shipit:

**Comment by dergigi** [comment]
Live now:
- https://opensats.org/blog/second-wave-of-education-grants
- [nostr post](https://njump.me/nevent1qqs09wlchfqhuj3z0lqgddkgtwq5tplw5sk46385qp2jtzw7rsxgjaspp4mhxue69uhkummn9ekx7mqpzemhxue69uhhyetvv9ujuurjd9kkzmpwdejhgqg5waehxw309aex2mrp0yhxgctdw4eju6t0qgs8suecw4luyht9ekff89x4uacneapk8r5dyk0gmn6uwwurf6u9rus52lrxc)

**Pull request merged**